homeassistant/components/hue/v2/light.py

```diff
@@ -39,7 +39,7 @@
     normalize_hue_transition,
 )
 
-EFFECT_NONE = "None"
+EFFECT_NONE = "none"
```

Comment:
why not keep this as `None` to avoid a breaking change?

Reply:
In order to translate `None` in other languages. It's strange to see a "onderwater, glinster, None"

Reply:
but this is the key, right? Can't we have the key using uppercase on the strings.json too? Not pretty, but would avoid breaking.

Reply:
I _believe_ that we always have these lowercase because there are some things that do not support it. But other than that, I rather keep this constant than change how this works

Reply:
My suggestion was not to remove the constant, just keep it as it was before `EFFECT_NONE = "None"`, and use `None` as the key in strings.json
But I am not aware if there is any restriction to make them lowercase.